### PR TITLE
Add Logging to README.md.

### LINKED ISSUE #1898: Logging is missing from the ReadMe
It seems like we have a wrapper for the Logging API in Python. Therefore we should add the following to the ReadMe (just like for the other APIs):

- a link at the top of the ReadMe to the Logging API section
- a Logging API section with descriptions and links to the documentation
- an example of how to use the Logging API

### PR DESCRIPTION
**Pull request by daspecster**
@tseaver LMKWYT and if there's anything I should add.
/cc @dhermes

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -194,6 +196,28 @@ manage projects using this Client Library.
...
+.. _Logging API docs: https://cloud.google.com/logging/docs/
+
+.. code:: python
+
+    from gcloud import logging
+    client = logging.Client()
+    logger = client.logger('log_name')
+    logger.log_text("A simple entry")  # API call
+
+
```
Comment (tseaver) [inline README.rst]: Extra line?

**Comment by tseaver** [comment]
LGTM, even if you decide to ignore my nitpick.

**Comment by tseaver** [comment]
Hmm, maybe flesh the example out to show fetching entries?  E.g.:

```python
>>> entries, token = logger.list_entries()
>>> for entry in entries:
...   print entry.payload
A simple entry
```

**Comment by tseaver** [comment]
@daspecster reminder to close #1898 after merge (I think having `closes #1898` at the end of the first line of the commit message hides it from Github).

**Comment by tseaver** [comment]
LGTM.

**Comment by daspecster** [comment]
@tseaver looks like the `closes #1898` in the commit message worked!

**Pull request merged**